### PR TITLE
Change default vocabulary.

### LINKED ISSUE #344: Undefined terms need an optional default expansion (e.g., via `@vocab` in second `@context`)
```
_:c14n0 <https://w3id.org/traceability/#undefinedTermbar> "baz 123 123123123" .
```

should look like:

```
_:c14n0 <https://w3id.org/traceability/#undefined-term-bar> "baz 123 123123123" .
```

or something more readable.

### PR DESCRIPTION
**Pull request by OR13**
closes #344

### CODE REVIEW

File: docs/openapi/components/schemas/common/IntellectualPropertyRights.yml [hunk 1]
```diff
@@ -52,7 +52,7 @@ example: |-
     "intellectualPropertyRightsType": "Copyright",
     "intellectualPropertyRightsProduct": {
       "type": ["Product"],
-      "id": "ecommerce-marketplace.example.com/catalog/lawn-flamingos-357fc1f4",
+      "id": "https://ecommerce-marketplace.example.com/catalog/lawn-flamingos-357fc1f4",
```
Comment (OR13) [inline docs/openapi/components/schemas/common/IntellectualPropertyRights.yml:55]: surprisingly this value can be mutable when it is not an IRI ; )

We really need to just sign the bytes...

File: packages/traceability-schemas/package.json [hunk 2]
```diff
@@ -10,11 +10,12 @@
...
     "lint": "eslint . --fix",
     "test": "node --max-old-space-size=4096 'node_modules/.bin/jest'",
-    "regenerate": "npm i && node ./scripts/regenerate.js",
+    "regenerate": "node ./scripts/regenerate.js",
+    "count-issuer-dependent": "node ./scripts/count-issuer-dependent.js",
     "build:openapi": "node ./scripts/schemas-to-openapi.js",
     "build:context": "node ./scripts/openapi-to-context.js",
     "build:vocab": "node ./scripts/schemas-to-vocab.js",
-    "build": "npm run build:openapi && npm run build:context && npm run build:vocab"
+    "build": "npm run build:openapi && npm run build:context && npm run build:vocab && npm run regenerate && npm run count-issuer-dependent"
```
Comment (OR13) [inline packages/traceability-schemas/package.json:18]: We don't need the JSOn-LD tests, because they duplicate the logic that runs on install.

File: .gitignore [hunk 3]
```diff
@@ -15,3 +15,6 @@ docs/testsuite/jest-html-reporters-attach/result.js
 docs/testsuite/jest-html-reporters-attach/index.js
 
 packages/traceability-schemas/data/*.json
+
+
+traceability-tests
```
Comment (OR13) [inline .gitignore:20]: I'm going to start a separate test suite for vc data model v2.

**Review by nissimsan — APPROVED** [review]

File: docs/openapi/components/schemas/common/IntellectualPropertyRights.yml (hunk 1 above)
Comment (nissimsan) [reply docs/openapi/components/schemas/common/IntellectualPropertyRights.yml:55]: Ohhh, rookie-mistake!! Shame-shame-shame...

**Review by mkhraisha — APPROVED** [review]

**Review by BenjaminMoe — APPROVED** [review]
